test/plugin/test_in_tail.rb

```diff
@@ -48,14 +48,14 @@ def cleanup_directory(path)
   MULTILINE_CONFIG = config_element(
     "", "", {
       "format" => "multiline",
-      "format1" => "/^s (?<message1>[^\\n]+)(\\nf (?<message2>[^\\n]+))?(\\nf (?<message3>.*))?/",
+      "format1" => "/^s (?<message1>[^\\n]+)(\\nf (?<message2>[^\\n]+))?(\\nf (?<message3>.[^\\n]+))?/",
```

Comment:
This means this change has breaking compatibility?

Reply:
ah, good catch. added new const. https://github.com/fluent/fluentd/pull/2837/commits/e3079703f92a7c63d849950ce51ba2cc8d063a91

However, I think this change has a breaking change. 
This change can emit the data as unmatched_data which the previous version of fluentd ignores.